docs/oidc-client-ts.api.md

```diff
@@ -310,7 +310,7 @@ export class UserManager {
     // (undocumented)
     protected readonly _popupNavigator: PopupNavigator;
     // (undocumented)
-    querySessionStatus(): Promise<SessionStatus | null>;
+    querySessionStatus(navigatorParams?: IFrameWindowParams, args?: ExtraSigninRequestArgs): Promise<SessionStatus | null>;
```

Comment:
to restore previous behavior and to have it the same way as the API functions signinPopup, signinRedirect: lets combine here the args as well like:
`{ silentRequestTimeoutInSeconds, ...args }?: IFrameWindowParams & ExtraSigninRequestArgs`